components/suggest/src/rs.rs

```diff
@@ -181,11 +157,23 @@ impl Record {
     }
 }
 
+impl From<Record> for RemoteSettingsRecord {
```

Comment:
cargo clippy said impl `From` is preferred over `Into`

Reply:
Makes sense, Rust will automatically generate the `Into` if you define a `From`.